health-services/PassiveData/app/build.gradle

```diff
@@ -61,7 +65,7 @@ dependencies {
     // Wear
     implementation 'androidx.wear:wear:1.2.0'
     // Health Services
-    implementation "androidx.health:health-services-client:1.0.0-alpha02"
+    implementation 'androidx.health:health-services-client:1.0.0-SNAPSHOT'
```

Comment:
This will be beta01 right?

Reply:
Yes! Keeping as the SNAPSHOT for now so the thing builds.